tools/toolchains/arm.py

```diff
@@ -77,11 +77,20 @@ def __init__(self, target, notify=None, macros=None,
             raise NotSupportedException(
                 "this compiler does not support the core %s" % target.core)
 
-        self.check_c_lib_supported(target, "arm")
+        c_lib = "std"
+        if hasattr(target, "c_lib"):
```

Comment:
There seems to be a code repetition. Can a method in `mbed_toolchain.py` be created and used instead?